lib/app/modules/bulk_data/bulk_data_patient_export_sequence.rb

```diff
@@ -14,8 +14,26 @@ class BulkDataPatientExportSequence < SequenceBase
       requires :token
       conformance_supports :Patient
 
-      def check_export_kick_off(klass)
-        reply = export_kick_off(klass)
+      attr_writer :run_all_kick_off_tests
+
+      def check_export_kick_off(klass, search_params: nil)
+        @search_params = search_params
+
+        reply = export_kick_off(klass, search_params: search_params)
+
+        # Servers unable to support _type SHOULD return an error and OperationOutcome resource
+        # so clients can re-submit a request omitting the _type parameter.
+        if search_params.present? && reply.code > 400
```

Comment:
Should this be `>=`? Same thing in line 118, so maybe this should be a little helper method.